wordpress-mobile/gutenberg-mobile
# Fix link button on Rich Text

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1875
`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/20172

This PR updates the gutenberg ref to bring a fix on `isURL` that will fix the mentioned issue on the LinksUI Modal.

Gif showing the issue:

![link_button](https://user-images.githubusercontent.com/9772967/74145015-2d2e3200-4bfe-11ea-814e-c19da5bd416b.gif)

To test:

**To test:**
1. Select some text on a Paragraph block (or any rich text powered block)
2. Tap on the links button on the block's the action bar.
3. Check that the Modal with the links options show as expected.


PR submission checklist:
